ql/src/test/results/clientpositive/llap/dynamic_semijoin_reduction_3.q.out

```diff
@@ -582,7 +584,7 @@ STAGE PLANS:
         Reducer 4 <- Reducer 3 (SIMPLE_EDGE)
         Reducer 5 <- Reducer 3 (SIMPLE_EDGE)
         Reducer 6 <- Reducer 3 (SIMPLE_EDGE)
-        Reducer 7 <- Reducer 3 (CUSTOM_SIMPLE_EDGE)
```

Comment:
Interesting switch here - suspicious 1:1, because the hash function for regular group-by != hash function for bucketing, needs attention

Reply:
This is the INSERT branch of a merge statement. Bucketing key (`clustered by (a)`) is same that is present in the right outer join hence it seems fine.
```explain merge into acidTbl as t using nonAcidOrcTbl s ON t.a = s.a 
WHEN MATCHED AND s.a > 8 THEN DELETE
WHEN MATCHED THEN UPDATE SET b = 7
WHEN NOT MATCHED THEN INSERT VALUES(s.a, s.b)```

Reply:
In fact, the other branches (DELETE and UPDATE) did not change, which seems fine too.